asyncudp/__init__.py

```diff
@@ -67,7 +69,7 @@ async def create_socket(local_addr=None, remote_addr=None):
 
     loop = asyncio.get_running_loop()
     transport, protocol = await loop.create_datagram_endpoint(
-        _SocketProtocol,
+        partial(_SocketProtocol, queue_max_size=queue_max_size),
```

Comment:
No need to use functools if I'm not mistaken.

```suggestion
        lambda: _SocketProtocol(queue_max_size),
```